website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,221 @@
+---
+title: 'React Native 0.76 - New Architecture by default, React Native DevTools, Box Shadows and more'
+authors: [blakef, cipolleschi, frankcalise, gabrieldonadel]
+tags: [engineering]
+date: 2024-10-23
```

Comment:
Is this the planned release date?
Just asking to figure out if we should plan a headline in [This Week In React](https://thisweekinreact.com/newsletter) (send on Wednesdays, usually end of day)

Reply:
yes, that's the tentative release date, if everything goes according to our plans! :D